contrib/babelfishpg_tsql/antlr/TSqlParser.g4

```diff
@@ -3938,51 +3939,18 @@ spatial_coloncolon_methods
     : data_type colon_colon function_call
     ;
 
-// this is no longer used:
-xml_data_type_methods
-    : xml_value_method
-    | xml_query_method
-    | xml_exist_method
-    | xml_modify_method
+xml_method_call
+    : xml_method LR_BRACKET expression_list RR_BRACKET
```

Comment:
Have we verified that XML method calls will always have an argument? Because if there is function call which does not have an argument then we should replace `expression_list` with `expression_list?`

Reply:
replaced `expression_list` with `expression_list?` and handled function based number of arguments check seperately.